fhir-bulkimportexport-webapp/src/main/java/com/ibm/fhir/bulkcommon/Constants.java

```diff
@@ -45,6 +45,7 @@
     public static final String IMPORT_PARTITTION_WORKITEM = "import.partiton.workitem";
     public static final String IMPORT_PARTITTION_RESOURCE_TYPE = "import.partiton.resourcetype";
     public static final String IMPORT_FHIR_STORAGE_TYPE = "import.fhir.storagetype";
+    public static final String IMPORT_FHIR_IS_VALIDATION_ON = "import.fhir.validation";
```

Comment:
please update the documentation to explain the new value

Reply:
yes, I have not drafted the document for import yet, do have plan to add them. let me open an issue for this.

Reply:
https://github.com/IBM/FHIR/issues/809